profile-googlefonts/src/checks/googlefonts/name/version_format.rs

```diff
@@ -0,0 +1,61 @@
+use std::sync::LazyLock;
+
+use fontspector_checkapi::{prelude::*, testfont, FileTypeConvert};
+use read_fonts::TableProvider;
+use regex::Regex;
+use skrifa::string::StringId;
+
+#[allow(clippy::unwrap_used)]
+static VALID_VERSION_RE: LazyLock<Regex> =
+    LazyLock::new(|| Regex::new(r"Version\s0*[1-9][0-9]*\.\d+").unwrap());
+
+#[check(
+    id = "googlefonts/name/version_format",
+    rationale = "
+        
+        For Google Fonts, the version string must be in the format \"Version X.Y\".
+        The version number must be greater than or equal to 1.000. (Additional
+        information following the numeric version number is acceptable.)
+        The \"Version \" prefix is a recommendation given by the OpenType spec.
+    
+    ",
+    proposal = "https://github.com/fonttools/fontbakery/issues/4829",
+    title = "Version format is correct in 'name' table?"
+)]
+fn version_format(t: &Testable, _context: &Context) -> CheckFnResult {
+    let f = testfont!(t);
+    let mut problems = vec![];
+    let name = f.font().name()?;
+    let mut num_entries: u8 = 0;
+    for record in name.name_record().iter() {
```

Comment:
This is such a common thing to do we have an API method on the `TestFont` object to do it: `get_name_entry_strings(StringId::VERSION_STRING)`.